runtime/libffi/z/ffi64.c

```diff
@@ -928,6 +940,13 @@ ffi_prep_cif_machdep(ffi_cif *cif)
 	  if (type == FFI_TYPE_POINTER)
 	    struct_size += ROUND_SIZE ((*ptr)->size);
 	}
+  else if (type == FFI_TYPE_STRUCT_FF || type == FFI_TYPE_STRUCT_DD)
+    {
+    /* If the ffi_type is STRUCT with two floating point parameters of same type
+       they will be passed into available FPRs, logic to calculate bytes for the struct
+       should be same as normal struct. */
```

Comment:
* "of same" should be "of the same"
* "passed into" should be "passed in"
* the comma should be a period, with "Logic" starting the second sentence

Reply:
Addressed in https://github.com/eclipse-openj9/openj9/compare/c2018048ad37140e438aec106082576687394dd6..b6043edad9cc1c6681422ab75fac8bf576b3dde3